programs/survival/easier_renewable_sponge.sc

```diff
@@ -0,0 +1,29 @@
+__on_player_attacks_entity(player,entity)->(
+	schedule(0,'__add_dead_fish',entity)//if not the fish's health isn't 0 and I cant check if its dead or not
```

Comment:
I woudn't schedule call for every player kill - also check the entity type here

Reply:
ok, i'll fix that